### PR TITLE
Adding our API security - Firecracker project

### PR DESCRIPTION
**Pull request by yigblst**

### CODE REVIEW

File: _data/projects/Firecracker.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+name: Firecracker
+desc: CLI that maps business logic, anomaly decider, attacker and visualizer
+site: https://github.com/blst-security/firecracker
+tags:
+  - API-securrity
```
Comment (shiftkey) [inline _data/projects/Firecracker.yml:5]: typo:

```suggestion
  - API-security
```

File: _data/projects/Firecracker.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+  - Rust
+  - CLI
+upforgrabs:
+  name: up for grabs
+  link: https://github.com/username/project/labels/up%20for%20grabs
```
Comment (shiftkey) [inline _data/projects/Firecracker.yml:10]: At the moment there are no open issues here, so I'm not sure which label you plan to use:

```suggestion
  link: https://github.com/blst-security/firecracker/labels/help%20wanted
```

File: _data/projects/Firecracker.yml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+name: Firecracker
+desc: CLI that maps business logic, anomaly decider, attacker and visualizer
+site: https://github.com/blst-security/firecracker
+tags:
+  - API-securrity
+  - Rust
+  - CLI
+upforgrabs:
+  name: up for grabs
```
Comment (shiftkey) [inline _data/projects/Firecracker.yml:9]: Please update this with whatever label you wish to use:

```suggestion
  name: help wanted
```

**Review by shiftkey — COMMENTED** [review]
A few changes before this is ready to merge:

**Comment by yigblst** [comment]
Closed due to name change and reopened under a new name.

**Pull request closed without merging**